packages/emails/src/send-email.tsx

```diff
@@ -49,6 +55,66 @@ const getTransport = () => {
         });
       }
       break;
+    case "sendgrid":
+      {
+        const options = {
```

Comment:
Make sure you have prettier running to auto-fix the code style to match the rest of the codebase.